ui/components/ErrorPage.js

```diff
@@ -1,96 +1,93 @@
-    lineHeight: '2rem',
-    marginBottom: '2rem',
-  },
-  errMessage: {
-    fontWeight: '400',
-    fontSize: '1.5rem',
-    color: 'gray',
-    fontStyle: 'italic',
-    marginTop: '2.5rem',
-  },
-
-  discussionforumlink: {
-    color: theme.palette.type === 'dark' ? '#00B39F' : '#',
-  },
-
-  mesh: {
-    display: 'block',
-    margin: 'auto',
-    marginTop: '3.125rem',
-    maxWidth: '50%',
-    height: '45%',
-  },
-  helpMessage: {
-    marginTop: '5rem',
-    color: theme.palette.type === 'dark' ? '#fff' : '#222',
-  },
+const StyledBox = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? theme.palette.background?.default : '#fff',
+  padding: '2rem',
+  textAlign: 'center',
+  borderRadius: 4,
+  height: '100%',
 }));
-
-const customMessages = [
-  'Oh, no. Please pardon our meshy app.',
-  'Oops. Please excuse the mesh.',
-  'Things tend to get a bit meshy around here.',
-  'Please pardon our mesh.',
-  'How did this mesh happen?',
-  "Well, isn't this a mesh?",
-  'Yikes. Things are a mesh here.',
-];
-
 /**
  * CustomErrorMessage component is used to display a custom error message when a page is not found.
  * @returns {JSX.Element} JSX.Element
  */
-function CustomErrorMessage({ message }) {
-  const classes = styles();
-  const [customMessage, setCustomMessage] = useState(customMessages[0]);
+const CustomErrorMessage = ({ message }) => {
+  const theme = useTheme();
+  const [customMessage, setCustomMessage] = useState('Oh, no. Please pardon our meshy app.');
 
   useEffect(() => {
+    const customMessages = [
+      'Oh, no. Please pardon our meshy app.',
+      'Oops. Please excuse the mesh.',
+      'Things tend to get a bit meshy around here.',
+      'Please pardon our mesh.',
+      'How did this mesh happen?',
+      "Well, isn't this a mesh?",
+      'Yikes. Things are a mesh here.',
+    ];
     setCustomMessage(customMessages[Math.floor(Math.random() * customMessages.length)]);
   }, []);
 
   return (
-    <NoSsr>
-      <div className={classes.rootClass}>
-        <div className={classes.errorSection}>
-          <Typography variant="h1">
-            <div className={classes.message}>{customMessage}</div>
-          </Typography>
-          <Typography variant="h5">
-            <div className={classes.errMessage}> {message || 'Page does not exist.'}</div>
-          </Typography>
-        </div>
-        <img src="/static/img/service-mesh.svg" alt="service meshed" className={classes.mesh} />
-        <Typography variant="body1">
-          <p className={classes.helpMessage}>
+    <UsesSistent>
+      <NoSsr>
+        <StyledBox>
+          <Box>
+            <Typography variant="h1" sx={{ fontSize: '3rem', lineHeight: '2rem', mb: 2 }}>
+              {customMessage}
+            </Typography>
+            <Typography
+              variant="h5"
+              sx={{
+                fontWeight: 400,
+                fontSize: '1.5rem',
+                color: 'gray',
```

Comment:
```suggestion
                color: theme.palette.text.secondary
```

ensure you match this provided token in sistent with the live changes here that it works properly.